src/main/java/com/epam/page/object/generator/validators/DuplicateTitleInInnerSearchRulesValidator.java

```diff
@@ -42,6 +50,6 @@ public boolean isValid(SearchRule searchRule, ValidationContext validationContex
 
     @Override
     public String getExceptionMessage(SearchRule searchRule, ValidationContext validationContext) {
-        return "Title = " + searchRule.getTitle() + " are duplicated";
+        return "Title = '" + duplicate + "' are duplicated";
```

Comment:
SearchRule is not an immutable class. If the title of this class changes and the value of 'duplicate' variable becomes inconsistent.